### PR TITLE
refactor/1952 - Update MCO to AIRAC 2205

### LINKED ISSUE #1952: Update KMCO to AIRAC 2205
Update MCO to AIRAC 2205

### PR DESCRIPTION
**Pull request by MichaelDMcCracken**
Resolves #1952 
The purpose of this pull request is to update MCO to AIRAC 2205.

### CODE REVIEW

File: assets/airports/kmco.json [hunk 1]
```diff
@@ -598,39 +613,14 @@
...
-                ["FACTS", "EARRS"],
-                ["SAWZZ", "HANDD", "LEFTI"],
-                ["KNOST", "CIGAR*"]
-            ]
-        },
-        "MZULO2": {
-            "icao": "MZULO2",
+        "MZULO3": {
+            "icao": "MZULO3",
             "name": "Mizulo Two",
```
Comment (Fechulo) [inline assets/airports/kmco.json:618]: Three

File: assets/airports/kmco.json [hunk 2]
```diff
@@ -686,17 +676,65 @@
                 ["SRQ*"],
                 ["SZW*"]
             ]
+        },
+        "OSPRY1": {
+            "icao": "RDSOX1",
```
Comment (Fechulo) [inline assets/airports/kmco.json:681]: OSPRY1

File: assets/airports/kmco.json [hunk 3]
```diff
@@ -686,17 +676,65 @@
...
+                "REMIS": ["GDUNN", "KNOST", "REMIS"]
+            },
+            "draw": [
+                ["BLOSM", "OSPRY", "GDUNN", "KNOST"],
+                ["REMIS*", "KNOST", "CIGAR*"]
+            ]
+        },
+        "RDSOX1": {
+            "icao": "RDSOX1",
+            "name": "Red Sox Two",
```
Comment (Fechulo) [inline assets/airports/kmco.json:705]: One

**Review by Fechulo — CHANGES_REQUESTED** [review]

File: assets/airports/kmco.json (hunk 2 above)
Comment (MichaelDMcCracken) [reply assets/airports/kmco.json]: fixed

**Review by Fechulo — APPROVED** [review]

**Pull request merged**